docs/examples/vault-default-user/README.md

```diff
@@ -26,7 +26,15 @@ This example requires:
 4. The RabbitMQ admin credentials were already written to Vault to path `spec.secretBackend.vault.defaultUserPath` with keys `username` and `password` (by some cluster-operator external mechanism. The cluster-operator will never write admin credentials to Vault).
 5. Role `spec.secretBackend.vault.role` is configured in Vault with a policy to read from `defaultUserPath`.
 
-Run script [setup.sh](./setup.sh) to get started with a Vault server in [dev mode](https://www.vaultproject.io/docs/concepts/dev-server) fullfilling above requirements. (This script is not production-ready. It is only meant to get you started experiencing end-to-end how RabbitMQ integrates with Vault.)
+Run script [setup.sh](./setup.sh) to get started with a Vault server in [dev mode](https://www.vaultproject.io/docs/concepts/dev-server) fulfilling above requirements.
+[setup.sh](./setup.sh) assumes you are using namespace `examples`, which can be created by:
+
+```shell
+kubectl create ns examples
+```
+
```

Comment:
Given that rabbitmq manifest doesn't set a namespace:

https://github.com/rabbitmq/cluster-operator/blob/9b84a28ad887c86893f327951b88e3aded7c7578/docs/examples/vault-default-user/rabbitmq.yaml#L1-L4

Perhaps we should update the `kubectl apply` to include a flag like `--namespace examples`?

It is true that the `setup.sh` script expects the namespace `example`, however, it is not so clear in other parts of the example.

Reply:
> Given that rabbitmq manifest doesn't set a namespace:
> 
> https://github.com/rabbitmq/cluster-operator/blob/9b84a28ad887c86893f327951b88e3aded7c7578/docs/examples/vault-default-user/rabbitmq.yaml#L1-L4
> 
> Perhaps we should update the `kubectl apply` to include a flag like `--namespace examples`?
> 
> It is true that the `setup.sh` script expects the namespace `example`, however, it is not so clear in other parts of the example.

Yes sir, that was the reason that I only change the `setup.sh` namespace to `default`, I assumed that we just follow the `rabbitmq.yaml` since it will apply to the default namespace. Adding `--namespace examples` would help a lot. Thank you.